wiki/Performance_points/id.md

```diff
@@ -0,0 +1,131 @@
+<!-- Pennek also wrote a similar sub-article that may have small bits of extra info. If needed, find it here: https://github.com/ppy/osu-wiki/blob/be1fec96041971d2bace05eb216952a58b7cabbc/wiki/Performance_points/Performance_point_systems/en.md we should probably figure what to do with these two at some point-->
+
+# Performa Poin
+
+**Performa poin** (atau, **pp** untuk singkatannya) adalah sebuah metrik ranking yang bertujuan agar lebih relevan secara kontekstual dengan perkembangan pemain di osu !.
```

Comment:
```suggestion
**Poin performa** (atau disingkat "pp") adalah sebuah pengukuran ranking yang bertujuan untuk mempermudah menampilkan perkembangan pemain dalam osu! secara kontekstual.
```

Reply:
we'd use "Performance points" instead as suggested above